packages/kit/src/runtime/server/index.js

```diff
@@ -43,6 +44,39 @@ export async function respond(incoming, options, state = {}) {
 		locals: {}
 	};
 
+	if (options.method_override.enabled && request.method.toUpperCase() === 'POST') {
+		const { strategy = '', key: method_key = '', allowedMethods = [] } = options.method_override;
```

Comment:
No need to provide defaults — once config has gone through validation, whatever was specified as the default in [src/core/config/options.js](https://github.com/sveltejs/kit/blob/master/packages/kit/src/core/config/options.js) is guaranteed to be present if no value was provided

Reply:
Thanks for the review. I agree regarding `enabled`, will remove.

It looks like you're also proposing changing `key` to `parameter` and `allowedMethods` to `allowed`. Those make sense, especially if we're getting rid of the hidden input method.

And do we want to include supporting method overrides via headers in this PR, or save it for a future enhancement?

Reply:
For simplicity let's skip `header` for now, it'll be an easy thing to add later. Thank you!